packages/react-wildcat/src/staticServer.js

```diff
@@ -213,9 +215,11 @@ function start() {
 }
 
 function close() {
-    if (server && server.close) {
-        return server.close();
+    if (!server || !server.close) {
```

Comment:
I try to return the happy path at the end of the method, and bail at the earliest possible moment.